snuba/cli/migrations.py

```diff
@@ -21,12 +21,13 @@ def migrations() -> None:
 
 
 @migrations.command()
-def list() -> None:
+@click.option("--partition-id")
+def list(partition_id: Optional[int]) -> None:
```

Comment:
Should we require the partition when running a migration that impacts a storage that is partitioned ?